src/utils/helpers.ts

```diff
@@ -1,4 +1,4 @@
-const fireplaceRegex = /[0-9A-Fa-f]{12}/g;
+const fireplaceRegex = /[0-9a-f]{12}/g;
```

Comment:
I see the commit says `feat: change a bit the regex on mac address for correct error message`, but it's not clear to me why the message is correct if we disallow capital case in the fireplace hex ID?

Reply:
On my side I got only lowercase mac address so with the previous regex, it wasn't working for me.